actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -617,19 +636,20 @@ private void processUserRequestV3(Map<String, Object> userMap, String signupType
     userMap.put(JsonKey.IS_DELETED, false);
     Map<String, Boolean> userFlagsMap = new HashMap<>();
     userFlagsMap.put(JsonKey.STATE_VALIDATED, false);
-
-    userFlagsMap.put(
-        JsonKey.EMAIL_VERIFIED,
-        (Boolean)
-            (userMap.get(JsonKey.EMAIL_VERIFIED) != null
-                ? userMap.get(JsonKey.EMAIL_VERIFIED)
-                : false));
-    userFlagsMap.put(
-        JsonKey.PHONE_VERIFIED,
-        (Boolean)
-            (userMap.get(JsonKey.PHONE_VERIFIED) != null
-                ? userMap.get(JsonKey.PHONE_VERIFIED)
-                : false));
+    if (StringUtils.isEmpty(managedBy)) {
```

Comment:
If it is managed user, we can set emailVerified and phoneVerified to default values as false right?